pkg/machines/components/vmSnapshotsCreateModal.jsx

```diff
@@ -36,37 +36,30 @@ const _ = cockpit.gettext;
 
 const NameRow = ({ onValueChanged, dialogValues, onValidate }) => {
     return (
-        <>
-            <label className="control-label" htmlFor="name">
-                {_("Name")}
-            </label>
-            <FormGroup validated={dialogValues.validationError.name ? "error" : "default"}
-                fieldId="name"
-                helperText={dialogValues.validationError.name}>
-                <TextInput value={dialogValues.name}
-                    id="name"
-                    type="text"
-                    onChange={(value) => onValueChanged("name", value)}
-                    aria-label={_("Name input text")}
-                />
-            </FormGroup>
-        </>
+        <FormGroup validated={dialogValues.validationError.name ? "error" : "default"}
+            label={_("Name")}
+            fieldId="name"
+            helperText={dialogValues.validationError.name}>
+            <TextInput value={dialogValues.name}
+                id="name"
```

Comment:
You forgot to add "validated" props to this "TextInput", which makes the nice red underlining when there is a validaiton error.